From f3581f5d55d360724d67009e6004721f77d51fc6 Mon Sep 17 00:00:00 2001
From: Alexander Larsson <alexl@redhat.com>
Date: Fri, 19 Sep 2014 08:52:24 +0200
Subject: [PATCH 2/2] Fix build failure by adding -Wno-error=unused-value

I got this:
../json_tokener.c: In function 'json_tokener_parse_ex':
../json_tokener.c:219:36: error: right-hand operand of comma expression has no effect [-Werror=unused-value]
   ( ++(str), ((tok)->char_offset)++, c)
                                    ^
---
 Makefile.am.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am.inc b/Makefile.am.inc
index fec591b..4163179 100644
--- a/Makefile.am.inc
+++ b/Makefile.am.inc
@@ -1,2 +1,2 @@
-AM_CFLAGS = -Wall -Werror -Wno-error=deprecated-declarations -Wextra -Wwrite-strings -Wno-unused-parameter -std=gnu99 -D_GNU_SOURCE -D_REENTRANT
+AM_CFLAGS = -Wall -Werror -Wno-error=deprecated-declarations -Wextra -Wwrite-strings -Wno-unused-parameter -Wno-error=unused-value -std=gnu99 -D_GNU_SOURCE -D_REENTRANT
 
-- 
1.8.3.1

